README.md

```diff
@@ -8,11 +8,11 @@ Have a multi-message chat with your bot.
 1) Make a new `Conversation` instance aware of your robot:
     
    ```javascript
-   var switchBoard = new Conversation(robot);
+   var switchBoard = new Conversation(robot, type);
    ```
     
-This will register a custom listener allowing the instance to check all incoming messages. If the message comes
-from a user that we're having a conversation with, it will be processed as the next step in an ongoing Dialog.
+This will register a custom listener allowing the instance to check all incoming messages. Type parameter can take one of two values: `user` (default) or `room`. It defines is this conversation with whole room or with particular user only.
+If the message comes from a user(room) that we're having a conversation with, it will be processed as the next step in an ongoing Dialog.
```

Comment:
`It defines is this conversation` reads better as : `It defines if this conversation is`